### ISSUE TITLE
[cvpaper] CVPR2019 #394 Dynamic Recursive Neural Network

### ISSUE THREAD

**Issue by kazuwjnlab** [kazuwjnlab/cvpaper#385]
## \#394 [Dynamic Recursive Neural Network](http://openaccess.thecvf.com/content_CVPR_2019/papers/Guo_Dynamic_Recursive_Neural_Network_CVPR_2019_paper.pdf)
Qiushan Guo, Zhipeng Yu, Yichao Wu, Ding Liang, Haoyu Qin, Junjie Yan
### どんな論文か？
DNNの同じ構造のBlockを最適化操作により簡潔化するDynamic Recursive Neural Network (DRNN)を提案．Blockにゲートを追加し，動的に最適化回数を決定できる．また，従来のRNNには学習困難な問題点があり，提案手法はLoopy Variable Batch Normalization提案し，それによりvolatile gradientを安定させることが可能．
### 新規性
Recursive Neural NetworkにBlockゲートを使い，最適化回数をコントロールできるようにした．RNN学習困難に対し改善できるLoopy Variable Batch Normalizationの提案．
### 結果
パラメータサイズ及び計算時間コストを削減したモデルでCIFAR-10, ImageNet-1kなどにおいて従来の手法（例：ResNet）より高い精度を得た．
### その他（なぜ通ったか？等）
効率が高い(メモリーコスト及び計算時間コスト)手法の提案．従来のあらゆる認識用ネットワークに容易に適応できる．
![thumb](https://drive.google.com/uc?export=view&id=12LdIP_SNL02a1nrhgYDpLdc3zwsS5N9r)
Recursive nerual network,
